src/oasis-indexer/api.ts

```diff
@@ -282,19 +300,23 @@ export const useGetRuntimeAccountsAddress: typeof _f7 = (network, runtime, addre
         ...arrayify(axios.defaults.transformResponse),
         (data: generated.RuntimeAccount, headers, status) => {
           if (status !== 200) return data
-          return {
+          return groupAccountTokenBalances({
             ...data,
             address_eth: getEthAccountAddress(data.address_preimage),
             evm_balances: data.evm_balances?.map(token => {
               return {
                 ...token,
-                balance: token.balance ? fromBaseUnits(token.balance, token.token_decimals) : undefined,
+                balance: token.balance // TODO: why do we test this? According to the API it must be there
+                  ? fromBaseUnits(token.balance, token.token_decimals)
+                  : '0',
               }
             }),
             balances: data.balances?.map(token => {
               return {
                 ...token,
-                balance: token.balance ? fromBaseUnits(token.balance, token.token_decimals) : undefined,
+                balance: token.balance // TODO: why do we test this? According tothe API it must be there
```

Comment:
OK, but the previously used `undefined` was not a valid value for this type according to [the definition](https://github.com/oasisprotocol/explorer/blob/031268b6e311773d9666c29f552c283f07f1daf8/src/oasis-indexer/generated/api.ts#L951). So we need a better fallback.

As far as I understand, we can extend the auto-generated types using the "declare module" trick, but we can't redefine an existing field as potentially undefined, so we can't change this definition to allow it.

We could introduce a new field for our re-calculated balances, but since the old value would still be part of the record, then there would be no way to make sure that all our code uses the new value.

One way would be to simply drop the the whole record where the balance is missing. Would that be OK with you?

Reply:
OK, so for now, we will simply drop the balances with missing actual balances.  (That's better than showing 0, I guess... and `undefined` was not an option.)

Reply:
Unfortunately, `undefined` is not a valid fallback value according to the type definition...

Reply:
Ah i see what you mean. The type comes from typeof generated.useGetRuntimeAccountsAddress, not inferred from the transformed value

Reply:
>  The type comes from typeof generated.useGetRuntimeAccountsAddress, not inferred from the transformed value

Exactly. And I see no way of overriding that type with another type that allows an existing non-nullable field to be nullable.